int32 ID_R_SHOULDER_PITCH      = 16
int32 ID_L_SHOULDER_PITCH      = 12
int32 ID_R_SHOULDER_ROLL       = 17
int32 ID_L_SHOULDER_ROLL       = 13
int32 ID_R_ELBOW               = 19
int32 ID_L_ELBOW               = 15
int32 ID_R_HIP_YAW             = 6
int32 ID_L_HIP_YAW             = 0
int32 ID_R_HIP_ROLL            = 8
int32 ID_L_HIP_ROLL            = 2
int32 ID_R_HIP_PITCH           = 7
int32 ID_L_HIP_PITCH           = 1
int32 ID_R_KNEE                = 9
int32 ID_L_KNEE                = 3
int32 ID_R_ANKLE_PITCH         = 10
int32 ID_L_ANKLE_PITCH         = 4
int32 ID_R_ANKLE_ROLL          = 11
int32 ID_L_ANKLE_ROLL          = 5
int32 ID_HEAD_PAN              = 20
int32 ID_HEAD_TILT             = 21
int32 ID_R_ELBOW_YAW           = 18
int32 ID_L_ELBOW_YAW           = 14
int32 ID_R_WRIST_YAW           = 23
int32 ID_L_WRIST_YAW           = 24
int32 ID_R_GRIPPER             = 25
int32 ID_L_GRIPPER             = 26
int32 ID_HEAD_TILT_2           = 22

int32[] servo_numbers
float64[] positions